docs/requirements.txt

```diff
@@ -3,6 +3,6 @@ datetime
 eccodes
 numpy
 pandas
-sklearn
+scikit-learn
```

Comment:
> Oh! Good catch. I just read about this. Either works, but it is recommended to pip install using scikit-learn, but then the import in the code must remain as import sklearn. A bit confusing.

Yeah, confusing! It took me a while to figure out why it wasn't building the readthedocs for `pypromice.postprocess.csv2bufr` because of this.

> Also, after seeing the machine learning seminar, it is kind of funny we are installing this huge powerful library just to do linear regression.... but, I really like the implementation using scikit-learn. If we ever wanted to simplify the dependencies, the linear regression could be re-written using an existing package such as numpy or pandas.

I think we should keep `scikit-learn`, also from the perspective that it might be useful for more sophisticated processing in the future.